@xen-orchestra/lite/src/components/host/system/HostSystemResourceManagement.vue

```diff
@@ -35,6 +32,8 @@ const controllerMemory = computed(() => {
     return
   }
 
-  return formatSizeRaw(controlDomain.memory_dynamic_max, 2)
+  const controlDomainMemory = formatSizeRaw(controlDomain.memory_dynamic_max, 2)
+
+  return `${controlDomainMemory.value} ${controlDomainMemory.prefix}`
```

Comment:
I think you can use `formatSize` in s`ize.utils.ts`
```suggestion
  return formatSize(controlDomain.memory_dynamic_max, 2)```